src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/ComponentExampleBox.razor

```diff
@@ -29,15 +30,17 @@
                        IconName="BitIconName.Copy"
                        Title="Copy source">
             </BitIconButton>
-            
+
             <pre class="code">
-                <code class="html">@HTMLSourceCode</code>
+            <code class="html">@HTMLSourceCode</code>
```

Comment:
code formatting issue

Reply:
The code formatting is ok...If we apply the formatting here, the result will not be what we expect.

Reply:
I see, ok